C/include/c4Index.h

```diff
@@ -136,6 +138,16 @@ CBL_CORE_API bool c4coll_isIndexTrained(C4Collection* collection, C4String name,
     @return  A new `C4IndexUpdater` reference, or NULL if there's nothing to update. */
 CBL_CORE_API C4IndexUpdater* C4NULLABLE c4index_beginUpdate(C4Index* index, size_t limit, C4Error* outError) C4API;
 
+/**
+ * Return the name of this index.
+ **/
+CBL_CORE_API C4SliceResult c4index_getName(C4Index* index) C4API;
```

Comment:
The underlying c4Index::getName returns a `slice`, so this result type should be `C4String`, not `C4SliceResult`. (I.e. there's no need to copy the name and force the caller to release it.)

Reply:
Okay thanks :)